contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,73 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+/*
+* COLUMN PRIVILEGES
+*/
+
+CREATE OR REPLACE VIEW information_schema_tsql.column_privileges
+AS SELECT CAST(u_grantor.rolname AS sys.nvarchar(128)) AS "GRANTOR",
+    CAST(grantee.rolname AS sys.nvarchar(128)) AS "GRANTEE",
+    CAST(sys.db_name() AS sys.nvarchar(128)) AS "TABLE_CATALOG",
+    CAST(nc.nspname AS sys.nvarchar(128)) AS "TABLE_SCHEMA",
+    CAST(x.relname AS sys.sysname) AS "TABLE_NAME",
+    CAST(x.attname AS sys.sysname) AS "COLUMN_NAME",
+    CAST(x.prtype AS sys."varchar"(10)) AS "PRIVILEGE_TYPE",
+        CAST(CASE
+            WHEN pg_has_role(x.grantee, x.relowner, CAST('USAGE' AS text)) OR x.grantable THEN CAST('YES' AS text)
+            ELSE CAST('NO' AS text)
+        END AS sys.varchar(3)) AS "IS_GRANTABLE"
+   FROM ( SELECT pr_c.grantor,
+            pr_c.grantee,
+            a.attname,
+            pr_c.relname,
+            pr_c.relnamespace,
+            pr_c.prtype,
+            pr_c.grantable,
+            pr_c.relowner
+           FROM ( SELECT pg_class.oid,
```

Comment:
do you have any performance numbers on this? could you include a comparison of how column_privileges performs here compared to in sql server?

Reply:
Currently, the interns and myself do not have performance numbers. What tools do we need to get this?

Reply:
I ran the testing files against Client Statistics in SSMS. They ran in 2932 ms in Babelfish and around 2600 in sql server.